docs/src/tutorial/genomic-surveillance.rst

```diff
@@ -0,0 +1,158 @@
+Run the workflow using a genomic surveillance configuration
+===========================================================
+
+In this tutorial, you will replace the custom contextual data with meaningful focal data and use a configuration commonly found in genomic surveillance applications.
+
+.. contents:: Table of Contents
+   :local:
+
+Prerequisites
+-------------
+
+1. :doc:`custom-data`. These instructions will introduce the workflow configuration that is modified in this tutorial.
+
+Setup
+-----
+
+If you are not already there, change directory to the ``ncov`` directory:
+
+   .. code:: text
+
+      cd ncov
+
+Curate data from GISAID
+-----------------------
+
+.. warning::
+
+   These steps require a GISAID account. `Register <https://www.gisaid.org/registration/register/>`__ if you do not have an account yet. However, registration may take a few days. Follow `alternative data preparation methods <../guides/data-prep.html>`__ if you wish to continue this tutorial in the meantime.
+
+We will download a custom focal set from GISAID's EpiCoV database.
+
+1. Navigate to `GISAID <https://www.gisaid.org/>`__, **Login**, and go to **EpiCoV** > **Search**.
+
+   .. image:: ../images/gisaid-epicov-search.png
+      :width: 400
+      :alt: GISAID EpiCoV Search
+
+2. Select a custom set of records. As an example, this guide will look for all records that pass the following criteria:
+
+   1. From Idaho
+   2. Collected between ``2022-02-15`` and ``2022-03-15``
+   3. Has a complete genome
+   4. Has an exact collection date
+
+   .. image:: ../images/gisaid-select-records-idaho-highlighted.png
+      :width: 700
+      :alt: GISAID EpiCoV Search
+
+   .. note::
+
+      You can choose your own focal set here, but note that more records will take the workflow longer to run, and the GISAID web interface has a maximum of 5,000 records per download.
+
+      Due to the subsampling logic defined in the pre-written ``--configfile``, this tutorial works best when the focal set is geographically within in the USA.
+
+3. Select **Download** > **Input for the Augur pipeline** > **Download**.
+4. Extract by opening the downloaded ``.tar`` file in your file explorer. It contains two files: one ending with ``.metadata.tsv`` and another with ``.sequences.fasta``.
+5.  Move these extracted files to the ``data/`` directory in the ``ncov/`` directory.
+6.  Rename the files as appropriate, for example ``idaho.metadata.tsv`` and ``idaho.sequences.fasta``.
+
+Edit the workflow configuration file
+------------------------------------
+
+This time, you will make some modifications to the pre-written configuration file ``genomic-surveillance.yaml`` before running the workflow. The contents are copied below (comments excluded):
+
+.. code-block:: yaml
```

Comment:
There is a lot going on in this configuration now! I like how you've broken everything down below, @victorlin, but I wonder if we could break the configuration itself into smaller sections to tackle individually. For example, we could show the next `inputs` section, describe the additional `background_data` input and new Idaho inputs and also how we're able to use the pre-aligned sequences with the `aligned` entry point.

Then we could show the `builds` section and call out the new `subsampling_scheme` key and other bits the user may want to change.

Then we could show the `subsampling` section and go into the detail about what each component does.

Reply:
I like this idea! I've just been writing and trying to fill the gaps, not thinking about organization much. I'll make a change to address this.